src/main/java/com/example/demo/controllers/LocationController.java

```diff
@@ -0,0 +1,38 @@
+package com.example.demo.controllers;
+
+import com.example.demo.entities.Location;
+import com.example.demo.services.LocationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+
+@RestController("/api")
+@RequestMapping(value = "/api")
```

Comment:
```suggestion
@RequestMapping(value = "/api")
You will probably want to add the following to make sure your api uses json
```suggestion
@RequestMapping(value = "/api", produces = MediaType.APPLICATION_JSON)
```